src/diceBot/DoubleCross.rb

```diff
@@ -51,8 +51,8 @@ def getHelpMessage
   # 1. ダイス数
   # 2. クリティカル値
   # 3. 修正値
-  # 4. 達成値
-  DX_SHIPPU_DOTO_RE = /\A(\d+)DX(\d+)?([-+][-+\d]+)?(?:>=(\d+))?\z/io.freeze
+  # 7. 達成値
+  DX_SHIPPU_DOTO_RE = /\A(\d+)DX(\d+)?(([-+](\d+|\d+[\+\-\*]\d+|\(\d+([\+\-\*]\d+)*\)))+)?(?:>=(\d+))?\z/io.freeze
```

Comment:
```suggestion
  DX_SHIPPU_DOTO_RE = /\A(\d+)DX(\d+)?([-+][-+*\d]+)?(?:>=(\d+))?\z/io.freeze
```
`()` の計算は事前に処理されるのでここに含める必要はありません